share/spice/indego_bikeshare_locations/indego_bikeshare_locations.css

```diff
@@ -0,0 +1,63 @@
+.mapview-marker__icn.ddgsi-circle {
+  font-size: 6px;
+}
+
+.tile--indego_bikeshare_locations.tile--c--n {
+  width: 13em;
+  height: 13em;
+}
+
+.tile--indego_bikeshare_locations {
+    text-align: center;
+    height: 12em;
+}
+
+.tile--indego_bikeshare_locations .zci__header {
+    height: 2.5em;
+}
+
+.tile--indego_bikeshare_locations .side_by_side {
+    overflow: auto;
+    margin-left: auto;
+    margin-right: auto;
+    width: 100%;
+}
+
+.tile--indego_bikeshare_locations .side_by_side .left {
+    float: left;
+    border-right: 1px solid #E5E5E5;
+}
+
+.is-mobile .tile--indego_bikeshare_locations .side_by_side .left {
+    width: 50%;
+}
+
+.tile--citi_indego_bikeshare_locations .side_by_side .right {
+    float: right;
+}
+
+.is-mobile .tile--indego_bikeshare_locations .side_by_side .right {
+    width: 50%;
+}
+
+p.tx-clr--dk.tx--14 {
```

Comment:
@moollaza 

I can fix the typo on [line 35](https://github.com/duckduckgo/zeroclickinfo-spice/pull/2102#discussion-diff-38688251R35) but lines 31-42 look namedspaced to me already.  

Is [this](https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/share/spice/bike_sharing/citi_bike_nyc/bike_sharing_citi_bike_nyc.css) the final CSS for Citibike? If so, I can just copy it over and rename.